### PR TITLE
Fix microbuild signing configuration

### PR DESCRIPTION
**Pull request by natemcmaster**
Fix the file paths for code signing. cref https://github.com/aspnet/BuildTools/pull/766

### CODE REVIEW

**Comment by natemcmaster** [comment]
@HaoK this is the last one we need before we can merge https://github.com/aspnet/BuildTools/pull/766. Can you take a quick look?

File: src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj [hunk 1]
```diff
@@ -66,20 +66,19 @@
...
 
-    <!-- Binaries from partner teams that should be signed already -->
-    <ExcludePackageFileFromSigning Include="tools/$(TargetFramework)/any/NuGet.Frameworks.dll" Certificate="$(AssemblySigningCertName)" />
-    <ExcludePackageFileFromSigning Include="tools/$(TargetFramework)/any/System.Runtime.CompilerServices.Unsafe.dll" Certificate="$(AssemblySigningCertName)" />
+    <!-- Binaries from other projects that should be signed already -->
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.FileProviders.Physical.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.FileProviders.Physical.dll" />
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.FileSystemGlobbing.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.FileSystemGlobbing.dll" />
+    <ExcludePackageFileFromSigning Include="$(PublishDir)Microsoft.Extensions.Primitives.dll" PackagePath="tools/$(TargetFramework)/any/Microsoft.Extensions.Primitives.dll" />
```
Comment (HaoK) [inline src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj]: Is this a change from before where we used to sign all the other packages that got pulled in?

Comment (natemcmaster) [reply src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj]: Yes. So each repo will sign in batches now. Before, these files came from the FileSystem repo unsigned and ended up in this package. With MicroBuild signing, they will be signed before the Scaffolding repo builds. Each repo only needs to codesign the binaries it produces.

Comment (HaoK) [reply src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj]: Cool sounds much better

**Pull request merged**